### DISCUSSION TITLE
Software Engineer

### DISCUSSION THREAD

**Discussion by AFirouzi** [WeAreMahsaAmini/Core#192] [Introductions]
Hi All, 
I am Arvin,
Software engineer and Data analyst. Experienced in Python (Flask), C# (.NET Core)
I hope that I can be a help.

**Comment by fara-nak** [comment]
Hi, I am also a software engineer at Github, let me know if I can help.

**Comment by ehsaaanes** [comment]
Hi
I'm looking for people who can help creating an app for donation. 
I’d appreciate it if you check it here #163. 
please write a comment or let me know what you think about it. 
So much thanks
